src/rdf4cpp/util/Int128.hpp

```diff
@@ -0,0 +1,425 @@
+#ifndef RDF4CPP_INT128_HPP
+#define RDF4CPP_INT128_HPP
+
+#include <boost/multiprecision/cpp_int.hpp>
+#include <rdf4cpp/writer/BufWriter.hpp>
+#include <dice/hash.hpp>
+
+namespace rdf4cpp::util {
+#ifdef __SIZEOF_INT128__
+    using Int128 = __int128;
```

Comment:
`Int128` should not be in util namespace.

Just `rdf4cpp::Int128`.
Also for use in `xsd:decimal` define `using Decimal128 = BigDecimal<>` and use that instead